controllers/rcj_soccer_referee_supervisor/referee/event_handlers.py

```diff
@@ -10,15 +10,14 @@ def __init__(self):
 
     def handle(
         self,
-        supervisor,  # Supervisor from supervisor.py
+        referee,  # Referee from referee.py
```

Comment:
@Adman well, I guess we thankfully no longer have self-referencing issues here, so we could perhaps directly refer to the `Robot` type here, or am I completely off the rails?

Reply:
`referee` doesn't have any parent class. So we would once again ended up in circular import.